llvm/test/MC/BPF/insn-unit.s

```diff
@@ -61,6 +61,9 @@
 // CHECK-32: c3 92 10 00 00 00 00 00 	lock *(u32 *)(r2 + 16) += w9
 // CHECK: db a3 e2 ff 00 00 00 00 	lock *(u64 *)(r3 - 30) += r10
 
+  callx r2
+// CHECK: 8d 02 00 00 00 00 00 00 	callx r2
```

Comment:
I am waiting for a compile to finish, but I wanted to say that it looks right to me. The following is a test case from binutils:

```
  28:   8d 06 00 00 00 00 00 00         callr %r6
```

**Note**: The mnemonic will change.

Otherwise, everything looks good! Thanks again.